### ISSUE TITLE
Add hour information in dropbox file format

### ISSUE THREAD

**Issue by vzemlys** [cpfair/tapiriik#369]
I've experienced this problem when syncing from Garmin to dropbox. If I run twice a day in the same location, only one of the activities is synced, since both of the activities get the same name. The simple fix would be to change the following line

https://github.com/cpfair/tapiriik/blob/0e97b8eb46b9191f557d6fd46019223cc36e7de3/tapiriik/services/Dropbox/dropbox.py#L47

to 

```
ConfigurationDefaults = {"SyncRoot": "/", "UploadUntagged": False, "Format":"tcx", "Filename":"%Y-%m-%d_%H-%M-%S_#NAME_#TYPE"}
```

**Comment by bartholemeu** [comment]
Agree. I've discovered this is the cause of my issues as i do lots of run-swim-run etc.
-%S is probably overkill but hours and minutes yes.

**Comment by christofferholmstedt** [comment]
I have the same issue when using tapiriik online (no local instance). Just like you I assume it is due to filenames ending up the same and Tapirik does not overwrite when I run/exercise multiple times a day but I haven't seen any confirmation that this is the case. Have you? Tapirrik history only says that the activity is private and has not been synced due to that but it is syncing a lot of other private activities.

Workaround for now is to update the file format manually. Press "reconfigure" on the dropbox tab and then "setup". Change format to something like this:
```
/<YYYY>/<YYYY><MM><DD>T<HH><MIN>_<NAME>_<TYPE>
```

**Comment by dikjones** [comment]
Thank you. Thank you. Thank you. Full re-sync enabled with the addition of folders!
